src/Datepicker/index.web.js

```diff
@@ -269,6 +288,8 @@ const Datepicker = (props) => {
         style={currentStyle}
         strictParsing
         autoComplete={autoComplete}
+        showMonthDropdown
+        showYearDropdown
```

Comment:
Let's allow this prop `showMonthDropdown` and `showYearDropdown` to be configurable. Add this to https://github.com/CareLuLu/react-native-web-ui-components/blob/7d459f8348f6a3f6e33539ba91018a6cb3f019dc/src/Datepicker/index.web.js#L298 and set default to `true` https://github.com/CareLuLu/react-native-web-ui-components/blob/7d459f8348f6a3f6e33539ba91018a6cb3f019dc/src/Datepicker/index.web.js#L318.